HeatStroke/Services/OBJ/OBJFile.h

```diff
@@ -27,6 +27,17 @@ namespace HeatStroke
 			unsigned int m_uiPositionIndex;
 			unsigned int m_uiUVIndex;
 			unsigned int m_uiNormalIndex;
+
+			// Required to store in a map
+			bool operator<(const OBJVertex& other) const
```

Comment:
I coded up a solution for the unordered_map method to test the efficiency as I was creating the index buffer, but an ordered map worked significantly faster. The unordered_map needs methods created for hashing (which is where I ran into problems with efficiency when I tested with the boost library) and an equality check for collisions (which overloads operator==). Overloading the operator< for STD::map was the more legible and efficient approach than unordered_map, using tuple<int, int, int> as the key, or embedded maps (i.e. map<int, map<int, ...>)